app/views/escorts/confirm_cancel.html.slim

```diff
@@ -0,0 +1,18 @@
+= content_for :breadcrumbs do
+  - breadcrumbs_for_page root: true do
+    - breadcrumb detainee_breadcrumb(escort.detainee), escort_path(escort)
+
+.escort.per-page
```

Comment:
Is this CSS needed for this page? This CSS was specifically introduced to the PER summary page.